virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -96,6 +96,9 @@ This release adds new features and enhancements related to the following compone
 //CNV-34687 Release note: Generally available (GA) CPU Hotplug
 * Hot plugging virtual machines is now generally available. If a virtual machine cannot be hot plugged, the condition `RestartRequired` is applied to the VM. You can view this condition in the *Diagnostics* tab of the web console.
 
+//CNV-37545 RN: enhancement Specializing a Windows VM flow during VM creation
+* You can now select `sysprep` options when you create a Windows VM from an instance type. Previously it was necessary to set the `sysprep` options by customizing the VM after its creation.
```

Comment:
```suggestion
* You can now select `sysprep` options when you create a MicroSoft Windows virtyal machine (VM) from an instance type. Previously, you had to set the `sysprep` options by customizing the VM after its creation.
```

Reply:
Because this is release note, VM has already been clarified as virtual machine earlier in the assembly.